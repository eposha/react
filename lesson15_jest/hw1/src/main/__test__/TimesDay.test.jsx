import React from "react";
import { shallow } from "enzyme";
import TimesDay from "../TimesDay";

describe("<TimesDay/ >", () => {
  it("should display TimesDay", () => {
    const wrappedComponent = shallow(<TimesDay />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
