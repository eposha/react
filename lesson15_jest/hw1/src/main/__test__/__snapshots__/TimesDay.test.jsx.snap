// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimesDay/ > should display TimesDay 1`] = `
<div
  className="time__day"
>
  <div
    className="time__day-hour"
    key="1"
  >
    01:00
  </div>
  <div
    className="time__day-hour"
    key="2"
  >
    02:00
  </div>
  <div
    className="time__day-hour"
    key="3"
  >
    03:00
  </div>
  <div
    className="time__day-hour"
    key="4"
  >
    04:00
  </div>
  <div
    className="time__day-hour"
    key="5"
  >
    05:00
  </div>
  <div
    className="time__day-hour"
    key="6"
  >
    06:00
  </div>
  <div
    className="time__day-hour"
    key="7"
  >
    07:00
  </div>
  <div
    className="time__day-hour"
    key="8"
  >
    08:00
  </div>
  <div
    className="time__day-hour"
    key="9"
  >
    09:00
  </div>
  <div
    className="time__day-hour"
    key="10"
  >
    10:00
  </div>
  <div
    className="time__day-hour"
    key="11"
  >
    11:00
  </div>
  <div
    className="time__day-hour"
    key="12"
  >
    12:00
  </div>
  <div
    className="time__day-hour"
    key="13"
  >
    13:00
  </div>
  <div
    className="time__day-hour"
    key="14"
  >
    14:00
  </div>
  <div
    className="time__day-hour"
    key="15"
  >
    15:00
  </div>
  <div
    className="time__day-hour"
    key="16"
  >
    16:00
  </div>
  <div
    className="time__day-hour"
    key="17"
  >
    17:00
  </div>
  <div
    className="time__day-hour"
    key="18"
  >
    18:00
  </div>
  <div
    className="time__day-hour"
    key="19"
  >
    19:00
  </div>
  <div
    className="time__day-hour"
    key="20"
  >
    20:00
  </div>
  <div
    className="time__day-hour"
    key="21"
  >
    21:00
  </div>
  <div
    className="time__day-hour"
    key="22"
  >
    22:00
  </div>
  <div
    className="time__day-hour"
    key="23"
  >
    23:00
  </div>
</div>
`;
